src/host/main.cpp

```diff
@@ -453,20 +453,36 @@ int main(int argc, char** argv)
 
     if (*start)
     {
+      // Count members with public encryption key as only these members will be
+      // handed a recovery share.
+      // Note that it is acceptable to start a network without any member having
+      // a recovery share. The service will check that at least one recovery
+      // member is added before the service can be opened.
```

Comment:
Tested in `membership.py`, starting a CCF service with different combinations of members.